### ISSUE TITLE
Error using traits

### ISSUE THREAD

**Issue by uchagani** [thoughtbot/fishery#23]
Trying to use the new traits feature following the docs and I'm getting an error that `Property 'params' does not exist on type 'UserFactory'` and `Property 'sequence' does not exist on type 'UserFactory'`.  I'm fairly new to js/ts so it could be something stupid i'm doing.

```
import { Factory } from 'fishery';
import { Factories, User, UserTransientParams } from './types';
import * as faker from 'faker';

class UserFactory extends Factory<User, Factories, UserTransientParams> {
    admin(roleId: string) {
        return this.params({
            role: 'admin',
            roleId: roleId || `admin-${this.sequence()}`,
        });
    }
}


const userFactory = UserFactory.define(() => ({
    firstName: faker.name.firstName(),
    lastName: faker.name.lastName(),
    fullName: faker.name.firstName(),
    role: faker.lorem.word(),
    roleId: faker.lorem.word()
}));

```

**Comment by tiagodws** [comment]
I can second this. Looking at the published package, apparently it contains the code before the new traits.

How's the package being published? I feel like something went wrong there. Maybe one of the maintainers could take a look.

**Comment by uchagani** [comment]
@stevehanson

**Comment by stevehanson** [comment]
@uchagani @tiagodws my bad. I forgot to build before publishing. I just published 0.4.1 and confirmed that the traits are included. Thank you for bringing this to my attention!

**Comment by uchagani** [comment]
thanks @stevehanson this fixed it for me.  I have another issue (https://github.com/thoughtbot/fishery/issues/24) if you wouldn't mind looking at when you're free.  thank you.
